docs/source/starthere/intro.rst

```diff
@@ -80,67 +163,19 @@ Once you have :ref:`installed NeMo <installation>`, then you can run the code be
     import soundfile as sf
     sf.write("output_audio.wav", audio.to('cpu').detach().numpy()[0], 22050)
 
-You can learn more by about specific tasks you are interested in by checking out the NeMo :doc:`tutorials <./tutorials>`, or documentation (e.g. read :doc:`here <../asr/intro>` to learn more about ASR).
```

Comment:
Change this sentence: "You can learn . . ." to "To learn more about specific tasks that interest you, check out the NeMo :doc:`tutorials <./tutorials>` or documentation (for example, read :doc:`here <../asr/intro>` to learn more about ASR).